src/Microsoft.AspNetCore.Hosting/Startup/StartupExceptionPage.cs

```diff
@@ -267,10 +267,24 @@ private static void WriteException(Exception ex, StringBuilder builder, ref bool
                 }
             }
 
+            var exceptionMessage = HtmlEncodeAndReplaceLineBreaks(ex.Message);
+            var reflectionTypeLoadException = ex as ReflectionTypeLoadException;
+            if (reflectionTypeLoadException != null && reflectionTypeLoadException.LoaderExceptions.Length > 0)
+            {
+                var exceptionMessageBuilder = new StringBuilder(exceptionMessage);
+                exceptionMessageBuilder.Append("<br />LoaderExceptions:<ul>");
```

Comment:
Could you split these in separate Appends so it's a little easier to disambiguate?
```
.Append("<br>")
.Append("Loader Exception:")
.Append("<ul>");
```
Also space between Loader and Exceptions